angr/exploration_techniques/tracer.py

```diff
@@ -495,6 +495,94 @@ def _compare_addr(self, trace_addr, state_addr):
         else:
             raise AngrTracerError("Trace desynced on jumping into %#x, where no library is mapped!" % state_addr)
 
+    def _check_qemu_unicorn_desync(self, state: 'angr.SimState', trace_curr_idx, state_desync_block_addr):
+        """
+        Check if desync occurred because large blocks are split up at different instructions by qemu and unicorn. This
+        is done by reconstructing part of block executed so far from the trace and state history and checking if they
+        the same
+        """
+
+        from capstone import CS_GRP_CALL, CS_GRP_IRET, CS_GRP_JUMP, CS_GRP_RET # pylint:disable=import-outside-toplevel
```

Comment:
Can we move this line of import to the top of this file?

Reply:
Done.